### PR TITLE
Add NCSI support for Venice

### PR DESCRIPTION
**Pull request by aasboddu**
Added support for NCSI over the mac1 interface of BMC. NCSI is a DMTF spec which defines standard interface for BMC to manage NIC.

### CODE REVIEW

**Review by mahkurap — APPROVED** [review]

**Comment by ojayanth** [comment]
Add description , like quick summary on the PR

File: arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-congo.dts [hunk 1]
```diff
@@ -105,6 +105,14 @@
 	pinctrl-0 = <&pinctrl_rgmii0_default &pinctrl_rgmii0_driving>;
 };
 
+&mac1 {
```
Comment (supven01) [inline arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-congo.dts:108]: Has the NCSI tested yet on any of these platforms?

Comment (aasboddu) [reply arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-congo.dts:108]: yes tested on Congo.

**Comment by aasboddu** [comment]
> Add description , like quick summary on the PR

added the summary on PR.

**Review by supven01 — APPROVED** [review]

**Pull request merged**